hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceLogContentBodyTable.tsx

```diff
@@ -53,7 +74,7 @@ const InstanceLogContentBodyTable = memo(
                         const isValuesDifferent =
                             fileContent.logA.json[question.name] !==
                             fileContent.logB.json[question.name];
-
+                        const isImg = ['img', 'photo'].includes(question.type);
```

Comment:
This is not enough, we are using also 'image' line 273 in` InstanceFileContentRich.s`, basically we need to know all the field types that can contain an image

Reply:
I have `image` and `photo` in my parsing, not `img`. Maybe it would be a good idea to add `image` as well.